### PR TITLE
Add kubevirt_template module

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

This PR add new module `Kubevirt_template`.

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

kubevirt_template

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by pkliczewski — APPROVED** [review]

**Comment by pkliczewski** [comment]
shipit

File: lib/ansible/modules/cloud/kubevirt/kubevirt_template.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    - Use Openshift Python SDK to manage the state of KubeVirt templates.
+
+version_added: "2.8"
+
+author: KubeVirt Team (@kubevirt)
+
+options:
+    objects:
+        description:
+            - List of any valid API objects, such as a I(DeploymentConfig), I(Service), etc. The object
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/kubevirt/kubevirt_template.py:31]: You can make each sentence as new array element.

File: lib/ansible/modules/cloud/kubevirt/kubevirt_template.py [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+              will be created exactly as defined here, with any parameter values substituted in prior to creation.
+              The definition of these objects can reference parameters defined earlier.
+            - As part of the the list user can pass also I(VirtualMachine) kind. When passing I(VirtualMachine)
+              user must use Ansible strucutre of the paramters not the Kubernetes API structure. For more information
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/kubevirt/kubevirt_template.py]:
```suggestion
              user must use Ansible structure of the parameters not the Kubernetes API structure. For more information
```

File: lib/ansible/modules/cloud/kubevirt/kubevirt_template.py [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+              please take a look at M(kubevirt_vm) module and at EXAMPLES section, where you can see example.
+        type: list
+    display_name:
+        description:
+            - "A brief, user-friendly name, which can be employed by user interfaces."
+        type: str
+    description:
+        description:
+            - "A description of the template. Include enough detail that the user will understand what is being deployed
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/kubevirt/kubevirt_template.py]:
```suggestion
            - A description of the template.
            - Include enough detail that the user will understand what is being deployed...
```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Rest LGTM

File: lib/ansible/modules/cloud/kubevirt/kubevirt_template.py (hunk 1 above)
Comment (machacekondra) [reply lib/ansible/modules/cloud/kubevirt/kubevirt_template.py:31]: I think it's okay, like this, but I can change if you want. Other than that I've applied your suggestions, thank you!

**Comment by machacekondra** [comment]
bot_status

**Comment by mmazur** [comment]
+1

**Review by Akasurde — APPROVED** [review]
shipit

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
bot_status

**Comment by machacekondra** [comment]
@Akasurde Can you please merge?

**Pull request merged**